src/ident/AclIdent.cc

```diff
@@ -10,8 +10,6 @@
 
 #include "squid.h"
 
-#if USE_IDENT
```

Comment:
Please remove the remaining USE_IDENT cases: `git grep USE_IDENT`.

Reply:
Removed.